include/seqan3/range/decorator/gap_decorator.hpp

```diff
@@ -137,7 +137,7 @@ class gap_decorator
      *
      * \stableapi{Since version 3.1.}
      */
-    using unaligned_seq_type = inner_type;
+    using unaligned_sequence_type = inner_type;
```

Comment:
You need to also add a deprecated alias, i.e.

```cpp
//!\deprecated Please use seqan3::gap_decorator::unaligned_sequence_type instead.
using unaligned_seq_type SEQAN3_DEPRECATED_310 = unaligned_sequence_type;
```

Not sure if you need the doxygen comment, you can check if it actually does somthing

Reply:
I'll add it. thx. :)
How should I check if the doxygen comment is needed ?

Reply:
I would build it and check if the alias is shown (https://seqan3-git-fork-simonsasse-gapdecoratorrename-seqan.vercel.app/usr/html/classseqan3_1_1gap__decorator.html#a441d8562fac08376653af3325c4326f8)

If it's shown, it should have the `deprecated` comment

Reply:
It was neither in the user nor the dev documentation so I removed the oxygen comment. Then the documentation test from the CI failed because the member unaligned_seq_type was not documented. So I now added it back in. Let's see :)